### ISSUE TITLE
Upon migrating guest to remote server, password less ssh is permanent

### ISSUE THREAD

**Issue by jay-katta** [kimchi-project/kimchi#1063]
Upon migrating guest to remote server, password less ssh is permanent. 
Due to that, from terminal able to log on to the remote server with out prompting password

I think, upon migration, passwordless ssh has to revoke.

**Comment by danielhb** [comment]
This bug has been fixed for a while now.

**Comment by alinefm** [comment]
@danielhb the patch is pending on ML yet.
Have you already tested it? If so, please, reply on ML and I will get it upstream ASAP.

**Comment by danielhb** [comment]
Yeah, sorry about that. I remembered testing and approving the first version. This is the second:

[Kimchi-devel] [PATCH V2] [Kimchi] Issue #1063: Upon migrating guest to remote server, password less ssh is permanent

I'll get it tested (hopefully it'll be applicable to current master) when I have the opportunity.
